proposed/extended-coding-style-guide.md

```diff
@@ -215,6 +245,9 @@ the class name.
 The opening brace for the class MUST go on its own line; the closing brace
 for the class MUST go on the next line after the body.
 
+There MUST NOT be a blank line preceding a closing brace and the next line
+after the opening brace MUST NOT be a blank line.
```

Comment:
Should we rewrite this so that we have one sentence for the opening brace and one sentence for the closing brace to make this easier to follow?

Something like:
>Opening braces MUST be on their own line and MUST NOT be preceeded or followed by a blank line.
>
>Closing braces MUST be on their own line and MUST NOT be preceeded by a blank line.


Though I still think we should allow blank lines padding the class body.

```php
class Foo
{

    protected $property;

}

```